src/Client.php

```diff
@@ -55,14 +59,15 @@ class Client
      * @param string $apiKey
      * @param string $appId
      * @param string $mode
+     * @param string $tenant
      * @param array<mixed, mixed> $clientOptions
      */
-    public function __construct(string $apiKey, string $appId, $mode = 'production', array $clientOptions = [])
+    public function __construct(string $apiKey, string $appId, $mode = 'production', $tenant = 'pomelopay', array $clientOptions = [])
```

Comment:
Can we potentially add this in clientOptions array instead so that we don't break the constructor signature.

Reply:
Okay will change as part of clientOptions